devito/ops/node_factory.py

```diff
@@ -0,0 +1,214 @@
+from sympy import Eq, Add, Mul, Pow, Integer, Float, Rational
+from devito import Dimension
+from devito.symbolics import Macro
+from devito.types import Symbol, Indexed, Array
+
+from devito.ops.utils import namespace
+
+
+class OpsNodeFactory():
```

Comment:
OPS

Reply:
I misunderstood your previous comment. 
When you said: `OPS capital` you were referring to my comments, not the class name. But i thought it was the class name and changed it.